src/angular/core/option/option.module.ts

```diff
@@ -1,5 +1,6 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { SbbOptionHint } from '@sbb-esta/angular/core/option/option-hint';
```

Comment:
Here you would have to use a relative path
```suggestion
import { SbbOptionHint } from './option-hint';
```
and re format (otherwise `yarn baz` would generate a wrong entry)